### PR TITLE
Feature/external link icons

### PR DESCRIPTION
**Pull request by Preston-Daily**
# Deployment Checklist:

- [ ] I have generated the production migrations using a fresh database
      (`docker-compose down`, `docker-compose up`, and then generated migrations)

# Pull Request Description

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx [hunk 1]
```diff
@@ -61,18 +50,12 @@ const SIGWebsite = styled.a`
...
-/*
-const SIGDiscord: any = styled(SIGButton)`
-  background: #1166bb;
-  color: white;
-  &:hover {
-    color: white;
-    background: #0055aa;
+  &:hover #child{
+    transition: filter 0.17s ease-in;
+    filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(230deg) brightness(101%) contrast(102%);
```
Comment (ClayMav) [inline apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx]: move these properties to a different styled component, specifically `ExternalIconWebsite`

File: apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx [hunk 2]
```diff
@@ -159,8 +158,14 @@ const SIGDetailPane: React.FC<ISIGDetailPaneProps> = (props: any): any => {
         <SIGDesc>{sig.desc}</SIGDesc>
         <Row>
           <ButtonRow>
-            <SIGWebsite href={sig.website}>Website</SIGWebsite>
-            <SIGDiscord href={sig.discord}>Discord</SIGDiscord>
+            <SIGWebsite href={sig.website}>
+              Website
+              <ExternalIconWebsite id="child" src="https://cdn.mstacm.org/static/external-link.svg" />
```
Comment (ClayMav) [inline apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx]: For the url, try to emulate this code block from the footer
```typescript
<img alt="ACM Web Logo" src={`${config.CDN_URI}/static/web_footer.png`} />
```

File: apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx [hunk 3]
```diff
@@ -159,8 +158,14 @@ const SIGDetailPane: React.FC<ISIGDetailPaneProps> = (props: any): any => {
...
+            </SIGWebsite>
+            <SIGDiscord href={sig.discord}>
+              Discord
+              <ExternalIconDiscord src="https://cdn.mstacm.org/static/external-link.svg" />
```
Comment (ClayMav) [inline apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx]: Apply similar treatment to the url

File: apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx (hunk 1 above)
Comment (Preston-Daily) [reply apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx]: Functionality is lost when I do this with my current implementation.

**Pull request merged**